openstack/gaussdbnosql/v3/instance/DeleteInstance.go

```diff
@@ -0,0 +1,26 @@
+package instance
+
+import (
+	"net/http"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+func DeleteInstance(client *golangsdk.ServiceClient, instanceId string) (string, error) {
+	// DELETE https://{Endpoint}/v3/{project_id}/instances/{instance_id}
+	raw, err := client.Delete(client.ServiceURL("instances", instanceId), nil)
+	return extraJob(err, raw)
+}
+
+func extraJob(err error, raw *http.Response) (string, error) {
```

Comment:
maybe add wait function if job id is returned? and not return exact jobid from delete, just nil as usual